articles/overview/what-is-quantum-computing.md

```diff
@@ -12,45 +12,51 @@ uid: microsoft.quantum.overview.what
 
-Microsoft is developing a quantum computer based on topological qubits. We believe a topological qubit will be less impacted by changes in its environment, therefore reducing the degree of external error correction. Qubits are fragile by nature and become less coherent as they interact with their environment. Topological qubits feature increased stability and resistance to environmental noise, which means they can more readily scale and remain reliable longer.
+Microsoft is developing a quantum computer based on topological qubits. We believe a topological qubit will be less impacted by changes in its environment, therefore reducing the degree of external error correction. Topological qubits feature increased stability and resistance to environmental noise, which means they can more readily scale and remain reliable longer.
 
-## Quantum computing software – the Microsoft Quantum Development Kit
+## Quantum computing – a full hardware and software stack 
 
-Designing a quantum program to harness interference may sound like a daunting challenge.  The Microsoft Quantum Development Kit (QDK) has been been introduced to make quantum programming and algorithm development more accessible. Our high-level programming language, Q#, addresses the challenges of quantum programming; it is integrated in a software stack that enables a quantum algorithm to be compiled down to the primitive operations of a quantum computer.
+Microsoft's quantum program is unique in that we focus on scaling each and every component of the system to deliver real quantum impact. This comprehensive approach involves: 
+* building a quantum computer using reliable, scalable and fault-tolerant topological qubits, 
+* engineering a unique cryogenic control plane with extremely low power and heat dissipation, 
+* developing a complete software stack to enable programming the quantum computer and controlling the system at scale. 
 
-Up to a certain scale (number of qubits), quantum computing can be simulated on a classical computer. Using simulation, you can start to write quantum programs today for running on quantum hardware tomorrow.  
+The open source Quantum Development Kit (QDK) has been been introduced to make quantum programming and algorithm development more accessible. Our high-level programming language, Q#, addresses the challenges of quantum programming.  We designed Q# as a high-level quantum-focused programming language focused on algorithm and application development. The Q# compiler is integrated in a software stack that enables a quantum algorithm to be compiled down to the primitive operations of a quantum computer.  Up to a certain scale (number of qubits), quantum computing can be simulated on a classical computer. Using simulation, you can start to write quantum programs today for running on quantum hardware tomorrow.  We’ve also paired Q# with samples, libraries, and learning exercises to make it easy to begin quantum programming today. 
```

Comment:
It might be good to link words like "samples" in the final sentence here to the relevant resources. At the least, that helps with search discoverability.

Reply:
This article doesn't have links, but it does have a "next steps" section which points to the Getting Started, which puts the usage of the Samples in context for the new learner.  If I had a link to Samples, then I would want to have links for other things too.  I'd rather just keep this doc without links and rely on the "Next Steps".